src/com/jambit/caesarCodec.java

```diff
@@ -43,7 +44,16 @@ public String encrypt(String msg, String encryptionKey) {
         } else {
             ccKey = Integer.parseInt(keys[0]);
         }
-        return encryptionCaesarCipher(msg, ccKey, newCharSet);
+
+        String[] splitLine = splitLine(msg);
+        for (int i = 0; i < splitLine.length; i++) {
+            encryptedString.append(encryptionCaesarCipher(splitLine[i], ccKey, newCharSet));
+            if (i + 1 != splitLine.length) {
+                encryptedString.append("\n");
+            }
+        }
+
+        return encryptedString.toString();
     }
 
     public String decrypt(String msg, String encryptionKey) {
```

Comment:
encrypt and decrypt shares duplicate code. IntelliJ should mention this as well. Try to reduce duplicate code by calling a common method